src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -38,45 +61,51 @@ def ml_client(
     )
 
 
-@pytest.fixture()
+@pytest.fixture
 def remote_client() -> PFClient:
-    # enable telemetry for CI
-    with environment_variable_overwrite(TELEMETRY_ENABLED, "true"):
-        yield PFClient(
-            credential=get_cred(),
-            subscription_id="96aede12-2f73-41cb-b983-6d11a904839b",
-            resource_group_name="promptflow",
-            workspace_name="promptflow-eastus",
-        )
+    if not is_live():
+        yield get_pf_client_for_playback()
+    else:
+        # enable telemetry for CI
+        with environment_variable_overwrite(TELEMETRY_ENABLED, "true"):
+            yield PFClient(
+                credential=get_cred(),
+                subscription_id="96aede12-2f73-41cb-b983-6d11a904839b",
+                resource_group_name="promptflow",
+                workspace_name="promptflow-eastus",
```

Comment:
looks like we still have some test using eastus not canary

Reply:
offline synced, currently our tests are still in canary region, will apply a switch later